ci/test.yml

```
@@ -374,6 +374,15 @@ core persistence asan test:
     expire_in: 1 week
     when: always
 
+core hwi test:
+  stage: test
+  <<: *gitlab_caching
+  needs:
+    - core unix frozen debug build
+  script:
+    - nix-shell --run "git clone https://github.com/bitcoin-core/HWI.git"
+    - nix-shell --run "cd HWI && poetry install && poetry run ./test/test_trezor.py --model_t ../core/build/unix/trezor-emu-core /nix/store/zlcyqsxag8kznk68ra01l8aysaad4v96-bitcoin-22.0/bin/bitcoind"
```

Is absolute path for bitcoind really needed? 

```suggestion
    - nix-shell --run "cd HWI && poetry install && poetry run ./test/test_trezor.py --model_t ../core/build/unix/trezor-emu-core bitcoind"
```

---

I tested it again and it's working without it, so I will change it to `bitcoind` only. Not sure if it wasn't working or I just assumed it wouldn't.